security/functions.php

```diff
@@ -189,4 +189,20 @@ function rsssl_generate_random_string($length) {
 	}
 
 	return $randomString;
+}
+
+/**
+ * Wrapper for admin user renamed but user enumeration enabled check
+ * @return bool
+ */
+function check_admin_user_renamed_and_enumeration_disabled() {
+	// Check if rename-admin-user has been loaded, while user-enumeration hasn't been loaded
+	if ( function_exists('rsssl_username_admin_changed') && ! function_exists('rsssl_disable_user_enumeration')) {
+		if ( rsssl_username_admin_changed() !== false ) {
```

Comment:
Kan korter denk ik? 

if ( rsssl_username_admin_changed() ) {

Reply:
@hesseldj de integratie wordt alleen ingeladen als de optie ingeschakeld is, daarom checkt de code nu op die twee functies. De fix runt alleen als de admin username veranderd is maar user enumeration nog disabled. Zou dat nog aan kunnen passen naar de optiewaarde zelf. Anders krijg je issues wanneer de optie niet ingeschakeld is en de functie niet bestaat.